### PR TITLE
feat : list and ratiing added

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Please review my components created

### CODE REVIEW

File: component.html [hunk 1]
```diff
@@ -577,6 +577,103 @@ <h2 class="component-heading" >Typography</h2>
...
+  style="width: 100%; height: 300px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (kushalsourav) [inline component.html:672]: avoid using inline styles.

**Review by kushalsourav — APPROVED** [review]
looks good to merge,

File: style/List.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+.no-bullet-list{
+    list-style: none;
+}
+.ordered-list{
+    list-style-type: lower-alpha;
+}
+.unordered-list{
+    list-style-type: square ;
+}
+.noti-list {
```
Comment (aiman-ansari) [inline style/List.css:26]: give some meaningful name here

**Review by aiman-ansari — CHANGES_REQUESTED** [review]
Instead of noti-list either you can give proper name to it or change it to stacked-list 
Spacing is great as well your naming conventions are meaningful and easy. 
Looks good to merge.

File: style/Rating.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    position: relative;
+    cursor: pointer;
+  }
+  .rating-wrapper input {
+    display: none;
+  }
+  
+  .rating-wrapper label:before {
+    content: "\2605";
+    position: absolute;
```
Comment (pankaj-prog) [inline style/Rating.css:24]: keep indentation consistent

File: style/List.css (hunk 2 above)
Comment (pankaj-prog) [reply style/List.css:26]: +1

**Review by pankaj-prog — APPROVED** [review]
Everything in your code looks good to me. suggested some minor changes. 
And on the UI part, contrast ratio of your primary color with white text is too low which makes it hard to read. Look into it.

File: style/List.css (hunk 2 above)
Comment (yashodaAgarwal) [reply style/List.css:26]: okay sure

File: style/Rating.css (hunk 3 above)
Comment (yashodaAgarwal) [reply style/Rating.css:24]: sure

**Pull request merged**